File: model/src/test/java/no/nav/pensjon/vtp/testmodell/repo/impl/TestscenarioTemplateLoaderTest.java
package no.nav.pensjon.vtp.testmodell.repo.impl;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.Test;

public class TestscenarioTemplateLoaderTest {
    @Test
    public void verify_that_all_scenarios_loads() {
        assertThat(new TestscenarioTemplateLoader().load())
                .isNotEmpty();
    }
}

File: model/src/test/java/no/nav/pensjon/vtp/testmodell/repo/impl/TestscenarioTemplateLoaderTest.kt
package no.nav.pensjon.vtp.testmodell.repo.impl

import org.assertj.core.api.Assertions
import org.junit.Test

class TestscenarioTemplateLoaderTest {
    @Test
    fun verify_that_all_scenarios_loads() {
        Assertions.assertThat(TestscenarioTemplateLoader().load())
                .isNotEmpty
    }
}
